cli/cmd/models.go

```diff
@@ -13,13 +22,111 @@ var modelsCmd = &cobra.Command{
 	Long: `Manage models, providers, and backends configured in LlamaFarm.
 
 Available commands will include listing models, testing inference, and syncing configs.`,
-	Hidden: true,
+	Hidden: false,
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("LlamaFarm Models Management")
 		cmd.Help()
 	},
 }
 
+var modelsListCmd = &cobra.Command{
+	Use:   "list [namespace/project]",
+	Short: "List available models for a project",
+	Long: `List all configured models for a LlamaFarm project.
+
+Examples:
+  # List models for explicit project
+  lf models list my-org/my-project
+
+  # List models from current directory config
+  lf models list`,
+	Run: func(cmd *cobra.Command, args []string) {
+		var ns, proj string
+
+		// Parse explicit project if provided
+		if len(args) >= 1 && strings.Contains(args[0], "/") {
+			parts := strings.SplitN(args[0], "/", 2)
+			ns = strings.TrimSpace(parts[0])
+			proj = strings.TrimSpace(parts[1])
+		}
+
+		cwd := getEffectiveCWD()
+		StartConfigWatcherForCommand()
+
+		// Resolve server configuration
+		serverCfg, err := config.GetServerConfig(cwd, serverURL, ns, proj)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+			os.Exit(1)
+		}
+		serverURL = serverCfg.URL
+		ns = serverCfg.Namespace
+		proj = serverCfg.Project
+
+		// Ensure server is up
+		ensureServerAvailable(serverURL, true)
+
+		// Call API endpoint
+		url := fmt.Sprintf("%s/v1/projects/%s/%s/models", strings.TrimSuffix(serverURL, "/"), ns, proj)
```

Comment:
you should re-use the fetchAvailableModels() function defined elsewhere

Reply:
even the types defined over in tui_chat.go are already defined. You're defilning them again here. Probably should move that stuff into the same place